pyanaconda/modules/storage/devicetree/viewer.py

```diff
@@ -613,6 +613,15 @@ def get_mount_point_constraints(self):
                     constraint.recommended = True
                 else:
                     constraint.required = True
+                # On MBR BIOS boot partition (typically labelled as biosboot) is not required.
+                if p.fstype == "biosboot":
+                    stage1_disk = self.storage.bootloader.stage1_disk
```

Comment:
Does not the folllowing work? I see the variable in the bootloader class.

stage1_device = self.storage.bootloader.stage1_device

Reply:
True.

Reply:
Yes, that could work. I was thinking this will be thrown away anyway so wanted to keep it rather working, but we have tests so let's look at it.